internal/plugin/service/organization/address/organization_address_schema.go

```diff
@@ -57,7 +57,7 @@ func ResourceSchema() map[string]schema.Attribute {
 			},
 		},
 		"organization_id": schema.StringAttribute{
-			Description: "ID of the organization.",
+			Description: "ID of the root account of the organization.",
```

Comment:
It shouldn't take account id, organization id only https://api.aiven.io/doc/#tag/Billing/operation/OrganizationAddressCreate
Could you elaborate on this a bit more, please?